tommath.h

```diff
@@ -147,8 +147,10 @@ TOOM_SQR_CUTOFF;
 #   endif
 #endif
 
+#define MP_TYPE_BIT(type) (CHAR_BIT * sizeof(type))
+
 /* size of comba arrays, should be at least 2 * 2**(BITS_PER_WORD - BITS_PER_DIGIT*2) */
-#define PRIVATE_MP_WARRAY (1u << (((CHAR_BIT * sizeof(mp_word)) - (2 * MP_DIGIT_BIT)) + 1))
+#define PRIVATE_MP_WARRAY (1u << ((MP_TYPE_BIT(mp_word) - (2 * MP_DIGIT_BIT)) + 1))
```

Comment:
Just don't use MP_TYPE_BIT here since PRIVATE_MP_WARRAY will be moved to tommath_private.h later.